src/loops/simple_programs_with_repitition/easy/CountItUp.java

```diff
@@ -0,0 +1,25 @@
+package loops.simple_programs_with_repitition.easy;
+
+/*
+Write a program that will ask the user for a sentence and then a character and 
```

Comment:
I think we should update this question to specify if we want to ignore Case or not, because the solution code does, but the code you wrote in your test does not.